flutter-for-react-native.md

```diff
@@ -0,0 +1,2007 @@
+---
+layout: page
+title: Flutter for React Native Developers
+permalink: /flutter-for-react-native/
+---
+
+This document is for React Native developers looking to apply their existing React Native knowledge to build mobile apps with Flutter. If you understand the fundamentals of the React Native framework, then you can use this document as a jump start to your Flutter development. 
+
+
+* TOC Placeholder
+{:toc}
+
+## A Brief Introduction to Dart for JavaScript Developers
+
+While React Native uses Javascript, Flutter uses a language called Dart. [Dart](https://www.dartlang.org/) is an open-source, scalable programming language, for building web, server, and mobile apps. It is an object-oriented, single inheritance language that uses a C-style syntax that is AOT-compiled into native and also transcompiles optionally into JavaScript. It supports interfaces, abstract classes and strong mode.
```

Comment:
"and strong mode" seems a bit weird. Maybe "and types"?

Reply:
Or "and strong types".

Also, spell it "JavaScript" consistently. And remove the comma at the end of "scalable programming language,"